pkg/apis/camel/v1/integration_types.go

```diff
@@ -70,6 +70,7 @@ type IntegrationStatus struct {
 // +kubebuilder:printcolumn:name="Phase",type=string,JSONPath=`.status.phase`,description="The integration phase"
 // +kubebuilder:printcolumn:name="Kit",type=string,JSONPath=`.status.kit`,description="The integration kit"
 // +kubebuilder:printcolumn:name="Replicas",type=integer,JSONPath=`.status.replicas`,description="The number of pods"
+// +kubebuilder:resource:path=integrations,scope=Namespaced,shortName=it,categories=kamel;camel
```

Comment:
ditto :)

Reply:
done, thanks :)

Reply:
The `kubebuilder:resource` marker already exists few lines above (67). Only the `categories` should be appended to the existing marker, instead of duplicating it.